data/repository/src/main/java/io/github/droidkaigi/confsched2020/data/repository/internal/DataContributorRepository.kt

```diff
@@ -4,32 +4,39 @@ import io.github.droidkaigi.confsched2020.data.api.DroidKaigiApi
 import io.github.droidkaigi.confsched2020.data.db.ContributorDatabase
 import io.github.droidkaigi.confsched2020.data.db.entity.ContributorEntity
 import io.github.droidkaigi.confsched2020.model.Contributor
-import io.github.droidkaigi.confsched2020.model.ContributorContents
 import io.github.droidkaigi.confsched2020.model.repository.ContributorRepository
 import javax.inject.Inject
+import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.map
 
 class DataContributorRepository @Inject constructor(
     private val api: DroidKaigiApi,
     private val contributorDatabase: ContributorDatabase
 ) : ContributorRepository {
-    override suspend fun contributorContents(): ContributorContents =
-        ContributorContents(contributorList())
+
+    override fun contributorContents(): Flow<List<Contributor>> =
+        contributorList()
 
     override suspend fun refresh() {
         val response = api.getContributorList()
         contributorDatabase.save(response)
     }
 
-    private suspend fun contributorList() = contributorDatabase
-        .contributorList()
-        .map { it.toContributor() }
+    private fun contributorList(): Flow<List<Contributor>> =
+        contributorDatabase
+            .contributorList()
+            .map {
+                it.mapIndexed { index, entity ->
+                    entity.toContributor(index + 1)
```

Comment:
I could have used `sort` value from API, but I felt it didn't matter for the purpose of displaying a non-zero index order.